unify-front-end/components/sanity/RichTextRenderer.tsx

```
@@ -701,11 +701,13 @@ export default function RichTextRenderer({
           );
         case 'normal':
         default:
-          // Add marginTop to first block to prevent text clipping
+          // Add marginTop to first block to prevent text clipping (only if not explicitly set by caller)
           const isFirstBlock = index === 0;
-          const finalBlockStyle = isFirstBlock
-            ? { ...blockStyle, marginTop: 8 }
-            : blockStyle;
+          const finalBlockStyle =
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)